index.js

```diff
@@ -31,8 +31,10 @@ import ganttCSS from "./js/webix/components/gantt/gantt.css";
 
 import kanban from "./js/webix/components/kanban/kanban.js";
 import pivot from "./js/webix/components/pivot/pivot.js";
-import querybuilder from "./js/webix/components/querybuilder/querybuilder.js";
 import report from "./js/webix/components/reports/reports.js";
+// import querybuilder from "./js/webix/components/querybuilder/querybuilder.js";
```

Comment:
if you want the `querybuilder` to be global then you might have to : 
```
const querybuilder = require("./path/here");
window.querybuilder = querybuilder;
```

The current `index.js` has some examples with the `.gantt` and `.reports`